extension/icu/scripts/inline-data.py

```diff
@@ -0,0 +1,27 @@
+import sys
+
+contents = bytearray(sys.stdin.buffer.read())
+
+# encode the data into the stubdata file
+result_text = ""
+first = True
+for byte in contents:
+	if first:
+		result_text += str(byte)
+	else:
+		result_text += "," + str(byte)
+	first = False
```

Comment:
```suggestion
result_text = ",".join(map(str, contents))
```

Reply:
Yeah that was bothering me too (scars from an old Python project where string concatenation was a performance bottleneck until we figured this out!) I was just trying to get @Mytherin 's existing scripts in with a minimum of modification to start. I'd also like to break up the lines to make diffing less stressful.